### PR TITLE
[4.12] remove disconnected jobs from arm64 job config

### PR DESCRIPTION
**Pull request by jinyunma**
Based on [slack discussion](https://redhat-internal.slack.com/archives/CF8SMALS1/p1728871441496139), update 4.12 arm64 job config

Removed following jobs from openshift-openshift-tests-private-release-4.12__arm64-nightly
- aws-ipi-disc-priv-sno-basecap-none-f60
- aws-ipi-disc-priv-sno-basecap-none-f60-destructive
- aws-ipi-disc-priv-sts-efs-f60
- aws-ipi-disc-priv-sts-efs-f60-destructive
- aws-ipi-disc-priv-sts-ep-f60
- aws-ipi-disc-priv-sts-ep-f60-destructive
- azure-ipi-disc-f60
- azure-ipi-disc-f60-destructive
- azure-ipi-disc-fullyprivate-f60
- azure-ipi-disc-fullyprivate-f60-destructive

Moved below jobs from 4.12__arm64-nightly to 4.12__amd64-nightly:
- aws-ipi-disc-priv-tp-f60
- aws-ipi-disc-priv-tp-f60-destructive

Removed following jobs from openshift-openshift-tests-private-release-4.12__arm64-stable
- aws-ipi-disc-priv-sts-ep-f60
- aws-ipi-proxy-sts-f60
- azure-ipi-disc-fullyprivate
- azure-ipi-fullyprivate-proxy-f60

### CODE REVIEW

**Comment by jinyunma** [comment]
cc @anpingli @jianlinliu @gpei @yunjiang29 to review

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__arm64-stable.yaml [hunk 1]
```diff
@@ -55,20 +55,6 @@ resources:
...
-      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:arm64-latest
-    env:
-      BASE_DOMAIN: qe.devcluster.openshift.com
-      COMPUTE_NODE_TYPE: m6g.xlarge
-      CONTROL_PLANE_INSTANCE_TYPE: m6g.xlarge
-      OCP_ARCH: arm64
-    test:
-    - chain: cucushift-installer-check-cluster-health
-    workflow: cucushift-installer-rehearse-aws-ipi-disconnected-private-cco-manual-security-token-service-private-s3-with-ep-sts-ec2-elb
 - as: aws-ipi-proxy-sts-f60
```
Comment (gpei) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__arm64-stable.yaml:58]: @jinyunma do we also need drop this one, this is actually being covered in the amd64 as well - https://github.com/openshift/release/blob/master/ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-stable.yaml#L81

Comment (jinyunma) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__arm64-stable.yaml:58]: I only updated jobs where the install workflow contains step `enable-qe-catalogsource-disconnected`, didn't touch others.
But I can update more jobs in arm64 nightly and arm64 stable based on latest discussion.

Comment (jinyunma) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__arm64-stable.yaml]: @gpei updated.

**Comment by jinyunma** [comment]
also cc @lwan-wanglin to review

**Comment by gpei** [comment]
looks good to me, leave it to @lwan-wanglin to confirm

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml [hunk 2]
```diff
@@ -243,6 +243,30 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-aws-ipi-disconnected-private-cco-manual-security-token-service-private-s3-with-ep-sts-ec2-elb
+- as: aws-ipi-disc-priv-tp-f60
```
Comment (yunjiang29) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml:246]: @jinyunma @jianlinliu @gpei do we need to update the frequency to f30? as the corresponding arm jobs got removed

Comment (gpei) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml:246]: seems the 4.12 is already in the extended support phase, only CVE fix will be available, please see the discussions in https://redhat-internal.slack.com/archives/GH9JHQFKN/p1728613664306739

**Comment by jianlinliu** [comment]
Per the slack thread discussion, sounds like we can drop all the 4.12 arm ci jobs for now, let us wait for a while for more thoughtful discussion to move forwards.

**Comment by lwan-wanglin** [comment]
> Per the slack thread discussion, sounds like we can drop all the 4.12 arm ci jobs for now, let us wait for a while for more thoughtful discussion to move forwards.

@jinyunma had discussed with me, she will continue this pr for disconnected jobs part, I will do the rest in another pr.

**Comment by jinyunma** [comment]
/pj-rehearse skip

cc @liangxia to review

**Pull request merged**